schultz_a07b.js

```diff
@@ -0,0 +1,93 @@
+/*Chris Schultz
+schultz_a07b.js
+Info 2124 WW
+Thoendel
+02-02-2020*/
+
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+
+//Step 1
+buildFoodsList (badFoods, "bread", "beer", "cheese", "shrimp", "tomatoes");   //Modified line as per Step 1   
+
+standardInput.on('data', function (text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    //Step 4 Begin
+    clearScreen();  //clearScreen is invoked regardless of next steps; let's do it first
+    if (text.toUpperCase() == "Q")  {//if input == Q || q....
+        console.log("Bye"); //...write bye to the screen...
+        process.exit(); //...and exit
+    }//end if
+    else if (text.toUpperCase() == "V") {  //if input == v || V....
+        console.log(displayList(groceryItems));  //display the list
+    }//end else if
+    else { //Otherwise!...(remember we already cleared the screen)
+        if (checkItem(text,badFoods) == false) { //...if user input is not in the bad foods list
+            groceryItems.push(formatItem(text)); //push the formatted user input into the array
+        }//end nested if
+        else if (checkItem(text,badFoods) == true) { //if and only if user input is in the bad foods list, print error

```

Comment:
you wouldn't need to check this again since you already know what was returned was not false - it must be true.